scripts/tests/calibnet_wallet_check.sh

```diff
@@ -71,6 +71,17 @@ done
 # wallet list should contain address two with transfered FIL amount
 $FOREST_WALLET_PATH list
 
+# wallet delete tests
+ADDR_DEL=$(forest-cli wallet new)
+
+forest-cli wallet delete -a "$ADDR_DEL"
+
+WALLET_OUT=$(forest-cli wallet list)
+
+if [[ ${WALLET_OUT} == *"$ADDR_DEL"* ]];then
+    echo "Error: the created wallet address $ADDR_DEL was not deleted"; exit 1;
+fi
```

Comment:
```suggestion
# Validate that the wallet no longer exists.
forest-cli wallet list | grep --null-data --invert-match "${$ADDR_DEL}"
```

Reply:
I get a
```
zsh: bad substitution
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', library/std/src/io/stdio.rs:1019:9                                                            
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[1]    1736333 exit 101   forest-wallet list
``` 

on my local

and on ci: `./scripts/tests/calibnet_wallet_check.sh:80:59: error: To expand via indirection, use arrays, ${!name} or (for sh only) eval. [SC2082]` - not sure what's the fix in this case.

Reply:
Sorry, it's `"${ADDR_DEL}"` at the end.